alpha/scripts/utils/updateKuserRegisterInfoGroupId.php

```diff
@@ -0,0 +1,40 @@
+<?php
+require_once('/opt/kaltura/app/alpha/scripts/bootstrap.php');
+
+if ($argc < 3)
+{
+	die("Usage: php $argv[0] groupId kuserIdsFile <realrun | dryrun> \n");
+}
+
+$groupId = $argv[1];
+$kuserIdsFile = $argv[2];
+$dryrun = true;
+if($argc == 4 && $argv[3] == 'realrun')
+{
+	$dryrun = false;

```

Comment:
dry run needs to be set into kaltura statement so all database calls that changes state will be ignored.